src/pages/Lend/Filter/LendFilterMenu.vue

```diff
@@ -7,43 +7,72 @@
 				<kv-icon class="filter-toggle-chevron" name="large-chevron" />
 			</div>
 			<div id="filter-menu">
-				<filter-section-gender :filter-menu-open="filterMenuOpen" />
-				<filter-section-sort :default-sort-indices="defaultSortIndices"/>
+				<div id="filter-section-mobile-reset-all" class="filter-section hide-for-medium">
+					<ais-clear-refinements class="clear-all-container">
+						<div
+							class="filter-mobile-reset-all-link"
+							slot-scope="{ canRefine, refine }"
+							@click="clearAllRefinementsAndClose(refine)"
+						>
+							Reset all
+						</div>
+					</ais-clear-refinements>
+					<div class="filter-mobile-close-icon-wrapper" @click="hideFilterMenu">
+						<kv-icon class="filter-mobile-close-icon" name="x"/>
+					</div>
+				</div>
+				<filter-section-gender :filter-menu-open="filterMenuOpen" class="filter-section" />
+				<filter-section-sort :default-sort-indices="defaultSortIndices" class="filter-section" />
 				<filter-section-categories
+					class="filter-section"
 					:custom-categories="customCategories"
 					:selected-custom-categories="selectedCustomCategories"
 					@toggle-custom-category="toggleCustomCategory"
 				/>
-				<filter-section-location />
-				<filter-section-range-slider :filter-menu-open="filterMenuOpen" />
+				<filter-section-location class="filter-section" />
+				<filter-section-range-slider class="filter-section" :filter-menu-open="filterMenuOpen" />
 
 				<div id="filter-section-advanced" class="filter-section" @click="showAdvancedFilters">
 					Advanced Filters
 				</div>
-				<div id="filter-section-close" class="filter-section" @click="hideFilterMenu">
-					Close
-				</div>
+				<ais-state-results class="filter-section filter-show-loans-mobile-wrapper hide-for-medium">
```

Comment:
Do you really need to wrap this in state-results? You don't appear to be using anything that component provides...

Reply:
We need `nbHits` from `AisStateResults`